pyfhirsdc/converters/extensionsConverter.py

```diff
@@ -87,6 +88,16 @@ def get_checkbox_ext():
     )
 
 def get_enable_when_expression_ext(expression, desc = None ):
+    ## Regex to find ${} pattern and extract the content.
+    ## TODO discuss how expression should be built 
+    enable_expression = re.split("^\${([^}]+)}(.*)", expression)
+    if len(enable_expression)>1:
+        linkid = enable_expression[1]
+        linkid_attr = enable_expression[2]
+        expression = '%/resource.repeat(item).where(linkId={0}){1}'.format(linkid,linkid_attr)
+    else: expression = '%/resource.repeat(item).where(linkId={0})'.format(enable_expression[0])
+    
+    
```

Comment:
this won't work if there is many ${} or if it doesn't start with ${ , why not a simple replace ?
 
```suggestion
    expression = re.sub(pattern=r'\${([^}]+)}(\.[^+=<>! ]+)?', '%/resource.repeat(item).where(linkId=\\0)\\1', expression)
       
```

to be tested